### ISSUE TITLE
Duplicating keyword argument names violates ENFORCE

### ISSUE THREAD

**Issue by azdavis** [sorbet/sorbet#886]
#### Input

No sorbet.run link given because it crashes.

Related to #885.

```ruby
# typed: true
extend T::Sig
sig {params(x: Integer, x: Integer).returns(Integer)}
def foo(x:, x:)
  x
end
```

#### Observed output

```
Exception::raise(): namer/namer.cc:392 enforced condition i < ctx.owner.data(ctx)->arguments().size() has failed: (no message provided)

Backtrace:
sorbet::Exception::enforce_handler(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, int) (in librealmain.so) (Exception.h:47)
sorbet::namer::NameInserter::fillInArgs(sorbet::core::MutableContext, std::__1::vector<sorbet::ast::ParsedArg, std::__1::allocator<sorbet::ast::ParsedArg> >) (in libnamer.so) (namer.cc:392)
sorbet::namer::NameInserter::preTransformMethodDef(sorbet::core::MutableContext, std::__1::unique_ptr<sorbet::ast::MethodDef, std::__1::default_delete<sorbet::ast::MethodDef> >) (in libnamer.so) (namer.cc:560)
sorbet::ast::PostPonePreTransform_MethodDef<sorbet::namer::NameInserter, sorbet::core::MutableContext, true>::call(sorbet::core::MutableContext, std::__1::unique_ptr<sorbet::ast::MethodDef, std::__1::default_delete<sorbet::ast::MethodDef> >, sorbet::namer::NameInserter&) (in libnamer.so) (treemap.h:204)
sorbet::ast::TreeMapper<sorbet::namer::NameInserter, sorbet::core::MutableContext>::mapMethodDef(std::__1::unique_ptr<sorbet::ast::MethodDef, std::__1::default_delete<sorbet::ast::MethodDef> >, sorbet::core::MutableContext) (in libnamer.so) (treemap.h:293)
sorbet::ast::TreeMapper<sorbet::namer::NameInserter, sorbet::core::MutableContext>::mapIt(std::__1::unique_ptr<sorbet::ast::Expression, std::__1::default_delete<sorbet::ast::Expression> >, sorbet::core::MutableContext) (in libnamer.so) (treemap.h:666)
sorbet::ast::TreeMapper<sorbet::namer::NameInserter, sorbet::core::MutableContext>::mapClassDef(std::__1::unique_ptr<sorbet::ast::ClassDef, std::__1::default_delete<sorbet::ast::ClassDef> >, sorbet::core::MutableContext) (in libnamer.so) (treemap.h:281)
sorbet::ast::TreeMapper<sorbet::namer::NameInserter, sorbet::core::MutableContext>::mapIt(std::__1::unique_ptr<sorbet::ast::Expression, std::__1::default_delete<sorbet::ast::Expression> >, sorbet::core::MutableContext) (in libnamer.so) (treemap.h:672)
std::__1::unique_ptr<sorbet::ast::Expression, std::__1::default_delete<sorbet::ast::Expression> > sorbet::ast::TreeMap::apply<sorbet::core::MutableContext, sorbet::namer::NameInserter>(sorbet::core::MutableContext, sorbet::namer::NameInserter&, std::__1::unique_ptr<sorbet::ast::Expression, std::__1::default_delete<sorbet::ast::Expression> >) (in libnamer.so) (treemap.h:712)
sorbet::namer::Namer::run(sorbet::core::MutableContext, sorbet::ast::ParsedFile) (in libnamer.so) (namer.cc:812)
sorbet::realmain::pipeline::name(sorbet::core::GlobalState&, std::__1::vector<sorbet::ast::ParsedFile, std::__1::allocator<sorbet::ast::ParsedFile> >, sorbet::realmain::options::Options const&, bool) (in libpipeline.so) (pipeline.cc:731)
sorbet::realmain::pipeline::resolve(std::__1::unique_ptr<sorbet::core::GlobalState, std::__1::default_delete<sorbet::core::GlobalState> >&, std::__1::vector<sorbet::ast::ParsedFile, std::__1::allocator<sorbet::ast::ParsedFile> >, sorbet::realmain::options::Options const&, sorbet::WorkerPool&, bool) (in libpipeline.so) (pipeline.cc:780)
sorbet::realmain::realmain(int, char**) (in librealmain.so) (realmain.cc:426)
main (in sorbet) (main.cc:7)
start (in libdyld.dylib) + 1
0x00000002 (in sorbet)

fella.rb:4: Failed to process tree (backtrace is above) https://srb.help/1001
     4 |def foo(x:, x:)
     5 |  x
     6 |end

fella.rb: Exception naming file: fella.rb (backtrace is above) https://srb.help/1001
Errors: 2
```

#### Expected behavior

Error about duplicated argument names

**Comment by jez** [comment]
This overlaps heavily with #885.

**Comment by azdavis** [comment]
This was closed in #1326 🎉
